nightly-benchmark/dgx-benchmark-historic-runs.csv

```diff
@@ -19,7 +19,3 @@
 20201130,simple,0.5799180269241333,0.0595338902855331
 20201130,shuffle,24.354144883155822,0.46329367607976524
 20201130,rand_access,0.010492658615112305,0.001179463979499339
-20201130,simple,0.5958759546279907,0.02384114128906203
-20201130,shuffle,22.320822596549988,0.9106601352938861
-20201130,rand_access,0.010452866554260254,0.0008015916163072284
-20201130,anon_mean,102.93944501876831,3.4379303100271668
```

Comment:
I'm guessing we plan to capture this benchmark starting tomorrow since we only run once daily via a cron job on the DGX. Is that right?

Reply:
Correct